packages/dev/serializers/src/glTF/2.0/Extensions/KHR_texture_transform.ts

```diff
@@ -56,6 +66,10 @@ export class KHR_texture_transform implements IGLTFExporterExtensionV2 {
             if (babylonTexture.wAng !== 0) {
                 textureTransform.rotation = -babylonTexture.wAng;
                 transformIsRequired = true;
+
+                if (babylonTexture.uRotationCenter !== 0 || babylonTexture.vRotationCenter !== 0) {
+                    textureTransform.offset = AdjustOffsetForRotationCenter(babylonTexture);
+                }

```

Comment:
I think this will work if we can guarantee these properties are not being animated by KHR_animation_pointer, but it's not easy to determine if they are.

Reply:
For now, maybe log a warning when this happens?

Reply:
We also have to be a bit careful with the math. The spec itself has some issues and we have to make sure the order of operations matches correctly. See https://github.com/KhronosGroup/glTF/issues/1563.

Reply:
We don't yet support KHR_animation_pointer export, but I do see how this could be problematic once we do. 
A warning sounds good for now.

As for the math piece, I tried to take care with the order of operations & preserved the negative rotation we use to comply with glTF spec. I did some testing, but let me do a bit more to be sure.

Reply:
Tested more for peace of mind, math should be good. I'd add a visual test, but it might be too much for such an edge case like this?